app/helpers/breeze_buddy/smart_router_client.py

```
@@ -0,0 +1,442 @@
+                error_text = response.text
+                logger.error(
+                    "Smart Router release failed",
+                    extra={
+                        "call_sid": call_sid,
+                        "status_code": response.status_code,
+                        "error": error_text,
+                    },
+                )
+                return False
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)